apps/cookbook/src/app/showcase/link-showcase/link-showcase.component.html

```diff
@@ -54,3 +54,20 @@ <h2>New tab</h2>
   In a non-web context, i.e. an app, any link will most likely launch the device's browser and
   effectively open a new tab outside the app context.
 </p>
+
+<h2>Custom trailing-icon</h2>
+<p>
+  If you want to use a custom icon after the link instead of the external link icon then include the Kirby scss @mixin <code>utils.trailing-icon</code>  
+  trailing-icon with @include and the icon name of your choosing:
+</p>
+<p>
+  <code>@include trailing-icon(iconName)</code>
+  the class will now look for the icon in assets. The example below is using
+  <code>"navigation"</code>
+  as iconName. If you want to use an external svg, then you can input an URL instead of iconName.
+  View code to see examples.
+</p>
```

Comment:
Consider wording and readability:
```suggestion
<p>
  If you want to use a custom icon after the link instead of the external link icon then include the Kirby SCSS @mixin <code>utils.trailing-icon</code>  
  and the icon name of your choosing: <code>@include trailing-icon(iconName)</code>
</p>
<p>
  The example below is using
  <code>"navigation"</code>
  as iconName. If you want to use an external svg, then you can input an URL instead of iconName.
  View code to see examples.
</p>
```